radicle/src/cob/issue.rs

```diff
@@ -411,16 +411,18 @@ impl<'a> Issues<'a> {
         title: impl ToString,
         description: impl ToString,
         tags: &[Tag],
+        assignees: &Vec<ActorId>,
```

Comment:
This should take a slice: `&[ActorId]`, we shouldn't force the caller to have a vector at hand.

Reply:
Done in 1ad7c47aaa3417c0bc23a02efd5c2be2798c2bed